base/applications/mspaint/lang/zh-TW.rc

```diff
@@ -78,6 +78,7 @@ BEGIN
         MENUITEM "旋轉/翻轉...\tCtrl+R", IDM_IMAGEROTATEMIRROR
         MENUITEM "拉伸/傾斜...\tCtrl+W", IDM_IMAGESTRETCHSKEW
         MENUITEM "剪裁", IDM_IMAGECROP
+		MENUITEM "&Grayscale colors", IDM_IMAGEGRAYSCALE
```

Comment:
These MENUITEM are still have indentation problems, please fix in all translation files.